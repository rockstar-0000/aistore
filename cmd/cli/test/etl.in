// RUN k8s
curl https://raw.githubusercontent.com/NVIDIA/ais-etl/master/transformers/hello_world/pod.yaml -o init_spec.yaml -s // IGNORE
ais etl init spec --from-file="init_spec.yaml" --comm-type="hpush://" --name=$ETL_NAME-1 // IGNORE
ais etl init spec --from-file="init_spec.yaml" --comm-type="hpush://" --name=$ETL_NAME-2 --arg-type="fqn" // IGNORE
rm -f init_spec.yaml // IGNORE

echo "0123456789" > $OBJECT_1.txt
ais bucket create ais://$BUCKET_1 | awk '{print $1 " " $2}'
ais object put $OBJECT_1.txt ais://$BUCKET_1/object_1.txt
rm -f $OBJECT_1.txt // IGNORE

ais etl show details $ETL_NAME-1 | grep "COMMUNICATION TYPE"
ais etl show details $ETL_NAME-1 | grep "ARGUMENT"
ais etl object $ETL_NAME-1 ais://$BUCKET_1/object_1.txt -
ais etl stop $ETL_NAME-1

ais etl show details $ETL_NAME-2 | grep "COMMUNICATION TYPE"
ais etl show details $ETL_NAME-2 | grep "ARGUMENT"
ais etl object $ETL_NAME-2 ais://$BUCKET_1/object_1.txt -
ais etl stop $ETL_NAME-2