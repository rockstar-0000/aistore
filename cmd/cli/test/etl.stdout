^*.ais://$BUCKET_1*.created$
^PUT.*=> ais://$BUCKET_1/object_1.txt$

COMMUNICATION TYPE:  hpush://
ARGUMENT TYPE:
Hello World!
^ETL\[etl.*\]stopped$
COMMUNICATION TYPE:  hpush://
ARGUMENT TYPE: fqn
Hello World!
^ETL\[etl.*\]stopped$
